### PR TITLE
Document public API for scheduling scripts

### PR DESCRIPTION
**Pull request by chekunkov**

### CODE REVIEW

**Comment by krotkiewicz** [comment]
LGTM @chekunkov

File: scrapy-cloud.rst [hunk 1]
```diff
@@ -238,11 +238,9 @@ After you :ref:`deploy <deploy>` your project, you will see the ``hello.py``
 script right below the list of spiders, in the schedule box (on Scrapinghub
 dashboard).
 
-You can also setup periodic jobs to run the script or do it via the API.
+You can also setup periodic jobs to run the script or do it via the API. Here is an example of script scheduling via the API using ``curl``::
```
Comment (eliasdorneles) [inline scrapy-cloud.rst]: I think usage of the term `periodic jobs` here is misleading -- this is just a one-off job, not really a periodic/recurrent job, right?

File: scrapy-cloud.rst [hunk 2]
```diff
@@ -238,11 +238,9 @@ After you :ref:`deploy <deploy>` your project, you will see the ``hello.py``
...
 
-For running the script through the API, you need to use a private API (which may subject to change in the future). Here is an example using ``curl``::
```
Comment (eliasdorneles) [inline scrapy-cloud.rst]: Another question, isn't there a supported way of pushing jobs using either python-scrapinghub or python-hubstorage?
Shouldn't we document that here?

File: scrapy-cloud.rst (hunk 1 above)
Comment (chekunkov) [inline scrapy-cloud.rst]: setup periodic jobs to run the script **OR** do it via the API. example shows only only scheduling via API. what if I move "You can also setup periodic jobs to run the script." to the previous paragraph and keep only API example here?

File: scrapy-cloud.rst [hunk 3]
```diff
@@ -234,16 +234,25 @@ Here is a ``setup.py`` example for a project that ships a ``hello.py`` script::
         entry_points = {'scrapy': ['settings = myproject.settings']},
     )
 
-After you :ref:`deploy <deploy>` your project, you will see the ``hello.py``
-script right below the list of spiders, in the schedule box (on Scrapinghub
```
Comment (chekunkov) [inline scrapy-cloud.rst]: scripts are not shown under the list of spiders, is that intentional @krotkiewicz ?

**Comment by chekunkov** [comment]
Ready for the next review round @eliasdorneles

**Comment by eliasdorneles** [comment]
Looks great, thanks @chekunkov !

**Pull request merged**